chain/address/address.go

```diff
@@ -91,7 +91,7 @@ func (a Address) Bytes() []byte {
 func (a Address) String() string {
 	str, err := encode(Testnet, a)
 	if err != nil {
-		panic(err)
+		panic(err) // I don't know if this one is okay
```

Comment:
The only 2 cases where this can happen - when the network prefix deosn't match, or the procotol is unknown. Given that we'd first have to parse an address like this, this panic is probably fine

Reply:
Yeah, so we parse this in, and don't allow parsing Undefined addresses, so this *should* be good. I'd feel much better if we had fuzzing here. @Kubuxu could you put something together?